modules/nw-metallb-software-components.adoc

```diff
@@ -14,11 +14,18 @@ When the Operator starts an instance of MetalLB, it starts a `controller` deploy
 `controller`::
 The Operator starts the deployment and a single pod.
 When you add a service of type `LoadBalancer`, Kubernetes uses the `controller` to allocate an IP address from an address pool.
+Ensure you have the following entry in your `controller` pod logs:
++
+[source,terminal]
+----
+"event":"ipAllocated","ip":"172.22.0.201","msg":"IP address assigned by controller
+----
 
 `speaker`::
 The Operator starts a daemon set for `speaker` pods.
 By default, a pod is started on each node in your cluster.
 You can limit the pods to specific nodes by specifying a node selector in the `MetalLB` custom resource when you start MetalLB.
+If a `speaker` pod becomes unavailable, read the `speaker` pod logs.
```

Comment:
It is not when 'speaker' pod becomes unavailable customer should read to speaker logs. It should be "In case of failure read the logs speaker pod. If the speaker pod itself is not in running state, oc describe pod <pod name> -n <namespace> can give more details".
The likelihood of speaker pod being unavailable is very less.

Reply:
Addressed

Reply:
@asood-rh Could you PTAL?